lib/rouge.rb

```diff
@@ -52,7 +56,7 @@ def lexer_dir(path = '')
 load_relative 'rouge/regex_lexer'
 load_relative 'rouge/template_lexer'
 
-Dir.glob(lexer_dir('*rb')).each { |f| Rouge::Lexers.load_lexer(f.sub(lexer_dir, '')) }
+Rouge::Lexers.load_proxies 'cache/proxies.json' unless Rouge.load_all?
```

Comment:
I think we should resort to using an absolute path here for robustness.
```suggestion
unless Rouge.load_all?
  Rouge::Lexers.load_proxies File.expand_path('../cache/proxies.json', __dir__)
end
```

Reply:
Have made a method similar to the one for `lexer_dir`.